### PR TITLE
Update TA Swagger, remove numberoflanguagestodetect

### PR DESCRIPTION
**Pull request by DavidLiCIG**
Parameter has been deprecated.

### CODE REVIEW

File: specification/cognitiveservices/data-plane/TextAnalytics/v2.0/TextAnalytics.json [hunk 1]
```diff
@@ -105,14 +105,6 @@
             "schema": {
               "$ref": "#/definitions/BatchInput"
             }
-          },
```
Comment (dsgouda) [inline specification/cognitiveservices/data-plane/TextAnalytics/v2.0/TextAnalytics.json]: This could be a breaking change, please acknowledge

**Review by dsgouda — COMMENTED** [review]
Other than minor comment, LGTM

File: specification/cognitiveservices/data-plane/TextAnalytics/v2.0/TextAnalytics.json (hunk 1 above)
Comment (DavidLiCIG) [reply specification/cognitiveservices/data-plane/TextAnalytics/v2.0/TextAnalytics.json]: The service is still accepting this parameter but deprecation has been known for a while now and this parameter is no longer being "processed". If users are already using this parameter in their code then yes, it's a breaking change.

**Comment by DavidLiCIG** [comment]
Is there anything more I need to do?

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**